crates/boundless-market/src/contracts/artifacts/Offer.sol

```diff
@@ -20,13 +20,20 @@ struct Offer {
     /// @notice Time at which bidding starts, in seconds since the UNIX epoch.
     uint64 biddingStart;
     /// @notice Length of the "ramp-up period," measured in seconds since bidding start.
-    /// @dev Once bidding starts, the price begins to "ramp-up." During this time, the price rises each block until it reaches maxPrice.
+    /// @dev Once bidding starts, the price begins to "ramp-up." During this time, the price rises
+    /// each block until it reaches `maxPrice`.
     uint32 rampUpPeriod;
     /// @notice Timeout for the lock, expressed as seconds from bidding start.
-    /// @dev This is the deadline for the lock to expire.
+    /// @dev Once locked, if a valid proof is not submitted before this deadline, the prover can
+    /// be "slashed", which refunds the price to the requester and takes the prover stake.
+    /// Additionally, the fee paid by the client is zero for proofs delivered after this time.
+    /// Note that after this time, and before `timeout` a proof can still be delivered to fulfill
+    /// the request.
     uint32 lockTimeout;
```

Comment:
I think we should also note that if a never locked request is fulfilled before this deadline then they are eligible for payment, if after then not

Reply:
paymentTimeout could be a better name? Though that may miss the nuance that you can still get stake if fulfilling a previously locked request.. naming is hard

Reply:
I've clarified it a bit further in 0e83b9d8fc0c05928c53eab57ac034a9fe9c8a66